parachains/runtimes/assets/statemine/tests/tests.rs

```diff
@@ -302,3 +302,70 @@ fn test_that_buying_ed_refund_does_not_refund() {
 			assert_eq!(Assets::total_supply(1), ExistentialDeposit::get());
 		});
 }
+
+#[test]
+fn test_asset_xcm_trader_not_possible_for_non_sufficient_assets() {
+	ExtBuilder::<Runtime>::default()
+		.with_collators(vec![AccountId::from(ALICE)])
+		.with_session_keys(vec![(
+			AccountId::from(ALICE),
+			AccountId::from(ALICE),
+			SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) },
+		)])
+		.build()
+		.execute_with(|| {
+			// Create a non-sufficient asset
+			// We set existential deposit to be identical to the one for Balances first
+			assert_ok!(Assets::force_create(
+				RuntimeHelper::<Runtime>::root_origin(),
+				1,
+				AccountId::from(ALICE).into(),
+				false,
+				ExistentialDeposit::get()
```

Comment:
@joepetrowski 
ok, thank you, I guess, I understand now,

for these new tests `test_asset_xcm_trader_not_possible_for_non_sufficient_assets` I changed it to: `let minimum_asset_balance = 1_000_000_u128;` and remove usage of `ExistentialDeposit::get()` and makes much more sense

I guess, everywhere in those Trader tests, `ExistentialDeposit::get()` is used as a some "random" constant/value,
because `ExistentialDeposit` is just used for `pallet_balances::Config` and this is not related to the `pallet_assets::Asset::min_balance`,
using `ExistentialDeposit::get()` just brings confusion

I am going to remove all usage of `ExistentialDeposit::get()` from those tests, ok?

Reply:
yeah, that sounds good :+1:

Reply:
continue in following issue: https://github.com/paritytech/cumulus/issues/1915